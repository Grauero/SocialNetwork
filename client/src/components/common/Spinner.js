import React from 'react';

import spinner from './spinner.gif';

const Spinner = () => (
  <div>
    <img
      src={spinner}
      alt="Loading..."
      style={{
        width: '80px',
        margin: 'auto',
        display: 'block'
      }}
    />
  </div>
);

export default Spinner;
