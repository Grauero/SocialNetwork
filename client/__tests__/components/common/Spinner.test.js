import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Spinner from '../../../src/components/common/Spinner';

it('matches snapshot', () => {
  const component = shallow(<Spinner />);

  expect(toJSON(component)).toMatchSnapshot();
});
