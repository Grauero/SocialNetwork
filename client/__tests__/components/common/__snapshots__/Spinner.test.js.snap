// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <img
    alt="Loading..."
    src="test-file-stub"
    style={
      Object {
        "display": "block",
        "margin": "auto",
        "width": "200px",
      }
    }
  />
</div>
`;
